hearts/static/room_ui.js

```diff
@@ -67,7 +67,14 @@ function RoomUI(callbacks) {
       buttonHtml = ('<button id="pass_button">' +
         'Pass 3 cards ' + direction + '</button>');
       $('#status').html(buttonHtml);
-      $('#pass_button').click(this.passButtonClickCallback);
```

Comment:
Isn't this one line equivalent to the "that = this" four lines below? What's the reason for the change?

Reply:
Yes it is.  I changed it thinking that the callback for the click would do something else in addition to `passButtonClickCallback`.   I initially tried to hide the pass button here, but realized it didnt work here.  (Something about how javascript callbacks are asynchronous)

Reply:
I can change it back